listenbrainz/listenstore/timescale_utils.py

```diff
@@ -110,3 +112,54 @@ def recalculate_all_user_data():
                       (user_timestamps[user_name][0], user_timestamps[user_name][1]), time=0)
         except KeyError:
             pass
+
+
+def refresh_listen_count_aggregate():
+    """
+        Manually refresh the listen_count continuous aggregate.
+
+        Arg:
+
+          year_offset: How many years into the past should we start refreshing (e.g 1 year, 
+                       will refresh everything that is 1 year or older.
+          year_count: How many years from year_offset should we update.
+
+        Example:
+
+           Assuming today is 2022-01-01 and this function is called for year_offset 1 and
+           year_count 1 then all of 2021 will be refreshed.
+    """
+
+    timescale.init_db_connection(config.SQLALCHEMY_TIMESCALE_URI)
+
+    end_ts = int(datetime.now().timestamp()) - 31536000
```

Comment:
extract 31536000 in a variable like SECONDS_IN_A_YEAR?

Reply:
Fixed. Waiting for tests to pass, then will merge.